lib/sync_checker/formats/publication_check.rb

```diff
@@ -2,7 +2,12 @@ module SyncChecker
   module Formats
     class PublicationCheck < EditionBase
       def root_path
-        "/government/publications/"
+        case edition_expected_in_live.publication_type
```

Comment:
Might be better to use `Whitehall::UrlMaker.new.public_document_path(edition_expected_in_live)` so it would work for any other types that have a different url (if there are any). We could probably use it in the base class and then we wouldn't have to do this root path stuff.